src/utils/string.cljs

```diff
@@ -38,3 +39,12 @@
 (defn at-least-n-chars?
   [s n]
   (>= (count s) n))
+
+(defn get-initials
+  "Returns `n` number of initial letters from `s`, all uppercased."
+  [s n]
+  (let [words (-> s str string/trim (string/split #"\s+"))]
+    (->> words
+         (take n)
+         (map (comp string/upper-case str first))
+         (string/join))))
```

Comment:
Omittable parenthesis around `string/join`.

Reply:
Yeah, you noticed that! I prefer to use parenthesis on multiline args because the code is perfectly aligned, and it carries the same syntax from another Lisp I love, Emacs Lisp. But forget what I said, I'll stop being stubborn, it's in the CSG https://guide.clojure.style/#threading-macros-and-optional-parentheses